CHANGELOG.md

```diff
@@ -12,6 +12,8 @@
 - Accept `input` and `data` field for the payload of transaction-related RPC methods [#6094](https://github.com/hyperledger/besu/pull/6094)
 - Add APIs to set and get the min gas price a transaction must pay for being selected during block creation [#6097](https://github.com/hyperledger/besu/pull/6097)
 - TraceService: return results for transactions in block [#6086](https://github.com/hyperledger/besu/pull/6086)
+- New option `--min-priority-fee` that sets the minimum priority fee a transaction must meet to be selected for a block. [#6080](https://github.com/hyperledger/besu/pull/6080) [#6083](https://github.com/hyperledger/besu/pull/6083)
+- Implement new `miner_setMinPriorityFee` and `miner_getMinPriorityFee` RPC methods [#6080](https://github.com/hyperledger/besu/pull/6080)
```

Comment:
are these really related to mining?

Reply:
they are related to block creation. I think it is the same case of this: https://github.com/hyperledger/besu/pull/6027#discussion_r1368620223

Reply:
the rename from _miner_ to _block creation_ is on the todo list, but it is not trivial and should be done with care, it will introduce a lot of breaking changes and need for external tools to update. 
I suggest that for the moment we keep it as is, and we consider _miner_ a synonym of _block creation_